public/index.html

```diff
@@ -2,9 +2,12 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="/app.css">
     <title>11-clock</title>
   </head>
   <body>
-
+    <div class="parent">
+    <h1 class="time">Time</h1>
+</div>
```

Comment:
Run beautify, then remember to include a `script` tag with an `src` attribute pointing to the `app.js` in your project. Remember that just like CSS, nothing is magically loaded in.